### DISCUSSION TITLE
Accessing project ID via Webhook response [Jan 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4787] [Discord Threads]
> 🙋‍♂️ **Soldado**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### That wouldn't be possible currently. Raise a `Feature request` in Github issue.  
#  
> 👨‍🦲 **dstala**  
#### https://github.com/nocodb/nocodb/issues/4594  
#  
> 👨‍🦲 **dstala**  
#### created one here.  
#  
> 🙋‍♂️ **Soldado**  
#### Just for reference, you can add static variable in json, just send {    
#### "Project": "XXXXXX",   
#### "Date": "YYYYYYY",   
#### "Info": [   
#### {{ json data }}   
#### ]   
#### }  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1048418082865025054)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
